pages/en/contrib/style-guide.md

```diff
@@ -24,6 +25,152 @@ Style conventions for this document:
 
 ## General guidelines
 
+{% include important.html content="
+
+The rules described in this document are evolving, therefore not all of our
+code-base is following them yet. This is expected and OK. Our approach is to
+incrementally improve the parts that are changing most often, i.e. fix the
+coding style only as part of changes needed for a bug fix/new feature.
+
+" %}
+
+### Variable declarations
+
+Prefer to use immutable variables declared with `const` keyword:
+
+### Good
+
+```js
+const user = app.models.User;
+const saltWorkFactor = user.settings.saltWorkFactor;
+```
+
+#### Bad
+
+```js
+var user = app.models.User;
+var saltWorkFactor = user.settings.saltWorkFactor;
+```
+
+In the rare cases where you need to split variable declaration and
+initialization, prefer to use `let` over `var`:
```

Comment:
`let` currently has a small caveat in that using it in loop constructs causes V8 to deoptimize the function, i.e., for now you should write:
```js
for (var i = 0; i < 42; ++i) doit(i);
```
Instead of:
```js
for (let i = 0; i < 42; ++i) doit(i);
```
This will be fixed in node.js v8.x but probably won't be fixed in v4.x and v6.x.